build/images/Dockerfile.build.agent.coverage

```diff
@@ -35,10 +35,13 @@ LABEL description="The Docker image to deploy the antrea-agent with code coverag
 
 USER root
 
+RUN mkdir -p /tmp/coverage
```

Comment:
I think this can be moved to line 46 to make docker image build more efficiently.
Please check other updated Dockerfiles in this patch as well.